test-data/unit/check-inference.test

```diff
@@ -430,19 +431,6 @@ def f() -> T: pass
 def g(a: T) -> None: pass
 [out]
 
-[case testUnsolvableInferenceResult]
```

Comment:
This is a test involving void and is no longer relevant.  It no longer tests anything interesting that is untested elsewhere.